provisioning/device/src/ProvisioningTransportRegisterMessage.cs

```diff
@@ -3,43 +3,14 @@
 
 using Microsoft.Azure.Devices.Authentication;
 
-namespace Microsoft.Azure.Devices.Provisioning.Client.Transport
+namespace Microsoft.Azure.Devices.Provisioning.Client
 {
     /// <summary>
     /// Represents a provisioning registration message.
     /// </summary>
     public class ProvisioningTransportRegisterMessage
```

Comment:
I'm not sure why the name of the file and this class were out of sync, but I actually like ```ProvisioningTransportRegisterRequest``` more than ```ProvisioningTransportRegisterMessage```. This message is a registration request after all